Model/Site.hs

```diff
@@ -4,9 +4,13 @@ module Model.Site
     , createSite
     , destroySite
     , overSiteQuota
+    , siteBranded
     ) where
 
-import Import
+import Import hiding (on)
```

Comment:
I think it's easier to keep the imports in your head if we're consistent with hiding. In this case, I think using `E.on` is more obvious. That way, everything unqualified is from `Import`.